src/convertToObject.js

```diff
@@ -12,7 +12,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
```

Comment:
Не може бути в назві `object, array, string` і так далі
```suggestion
  const styles = {};
```